ci-operator/step-registry/hypershift/hostedcluster/destroy/hostedcluster/hypershift-hostedcluster-destroy-hostedcluster-commands.sh

```diff
@@ -32,27 +38,51 @@ else
 fi
 
 createdAt=`oc -n clusters get hostedclusters $CLUSTER_NAME -o jsonpath='{.metadata.annotations.created-at}'`
-if [ -z $createdAt ]; then
+if [ -z $createdAt ] && [ "${PLATFORM}" == "aws" ]; then
```

Comment:
how this if block is only specific to `aws` platform?

Reply:
Its blocked to debug the failed cluster by dptp team, since we don't have permission to do much debugging, its better to unblock the destroy for powervs platform.

Reply:
@dharaneeshvrd  just want to make sure you aware, for some broken clusters, the destroy command cannot properly remove them. Thus, we have another periodic job to remove those broken clusters every 24 hours. We have other periodics to remove AWS resources created more than certain numbers of hours too, as a backup.

Reply:
@dharaneeshvrd lets add that flow as well as a follow up PR